waffle/tests/test_decorators.py

```diff
@@ -102,3 +104,13 @@ def test_flag_must_be_inactive_and_not_redirect(self):
         Flag.objects.create(name='foo', everyone=True)
         resp = self.client.get('/flagged_view_with_invalid_redirect')
         self.assertEqual(200, resp.status_code)
+
+    def test_waffle_callable_decorated_function_only_called_if_called_twice(self):
```

Comment:
Why should it only be called if called twice?

Reply:
Because in the first time you call you create a function that, when run with no parameters, will execute with the parameters. The second time, you call this function